src/renderer/widgets/AddPureProxiedModal/model/add-pure-proxied-model.ts

```diff
@@ -0,0 +1,352 @@
+import { createEffect, createEvent, createStore, sample } from 'effector';
+import { delay, spread } from 'patronum';
+import { Event } from '@polkadot/types/interfaces';
+import { ApiPromise } from '@polkadot/api';
+import { UnsubscribePromise } from '@polkadot/api/types';
+
+import { Transaction, TransactionType } from '@entities/transaction';
+import { toAddress } from '@shared/lib/utils';
+import { walletSelectModel } from '@features/wallets';
+import { walletModel, walletUtils } from '@entities/wallet';
+import {
+  type MultisigAccount,
+  type ProxyGroup,
+  type NoID,
+  ProxyType,
+  AccountId,
+  PartialProxiedAccount,
+  ProxyVariant,
+} from '@shared/core';
+import { wrapAsMulti, wrapAsProxy } from '@entities/transaction/lib/extrinsicService';
+import { proxyModel, proxyUtils } from '@entities/proxy';
+import { networkModel } from '@entities/network';
+import { balanceSubModel } from '@features/balances';
+import { Step, TxWrappers, AddPureProxiedStore } from '../lib/types';
+import { addPureProxiedUtils } from '../lib/add-utils';
+import { formModel } from './form-model';
+import { confirmModel } from './confirm-model';
+import { signModel } from './sign-model';
+import { submitModel } from './submit-model';
+import { subscriptionService } from '@entities/chain';
+import { proxiesModel } from '@features/proxies';
+
+const stepChanged = createEvent<Step>();
+
+const flowStarted = createEvent();
+const flowFinished = createEvent();
+
+const $step = createStore<Step>(Step.NONE);
+
+const $addProxyStore = createStore<AddPureProxiedStore | null>(null);
+const $transaction = createStore<Transaction | null>(null);
+const $multisigTx = createStore<Transaction | null>(null);
+
+const $txWrappers = createStore<TxWrappers>([]);
+
+type GetPureProxyParams = {
+  api: ApiPromise;
+  accountId: AccountId;
+};
+const getPureProxyFx = createEffect(({ api, accountId }: GetPureProxyParams): Promise<AccountId> => {
+  return new Promise((resolve) => {
+    const pureCreatedParams = {
+      section: 'proxy',
+      method: 'PureCreated',
+      data: [undefined, toAddress(accountId, { prefix: api.registry.chainSS58 })],
+    };
+
+    let unsubscribe: UnsubscribePromise | undefined;
+
+    unsubscribe = subscriptionService.subscribeEvents(api, pureCreatedParams, (event: Event) => {
```

Comment:
Why not this?
```suggestion
    const unsubscribe = subscriptionService.subscribeEvents(api, pureCreatedParams, (event: Event) => {
```

Reply:
with const I can't call it inside callback